bbconf/config_parser/bedbaseconfig.py

```diff
@@ -205,6 +219,35 @@ def _init_qdrant_backend(self) -> QdrantBackend:
                 f"error in Connection to qdrant! skipping... Error: {err}", UserWarning
             )
 
+    def _init_qdrant_text_backend(self) -> QdrantBackend:
+        """
+        Create qdrant client text embedding object using credentials provided in config file
+
+        :return: QdrantClient
+        """
+
+        return QdrantBackend(
+            dim=384,
+            collection=self.config.qdrant.text_collection,
+            qdrant_host=self.config.qdrant.host,
+            qdrant_api_key=self.config.qdrant.api_key,
+        )
+
+    def _init_bivec_object(self) -> Union[BiVectorSearchInterface, None]:
+        """
+        Create BiVectorSearchInterface object using credentials provided in config file
+
+        :return: BiVectorSearchInterface
+        """
+
+        search_backend = BiVectorBackend(
+            metadata_backend=self._qdrant_text_engine, bed_backend=self._qdrant_engine
+        )
+        search_interface = BiVectorSearchInterface(
+            backend=search_backend, query2vec="sentence-transformers/all-MiniLM-L6-v2"
```

Comment:
For the future, what is the difference between backend and interface? Is interface the correct term here?

Reply:
@ClaudeHu